docs/web3-eth.rst

```diff
@@ -1765,3 +1765,79 @@ Example
     > "Mist/v0.9.3/darwin/go1.4.1"
 
 ------------------------------------------------------------------------------
+
+.. _eth-getProof:
+
+getProof
+========
+
+.. code-block:: javascript
+
+    web3.eth.getProof(address, storageKey, blockNumber, [callback])
+
+Returns the account and storage-values of the specified account including the Merkle-proof as described in `EIP-1186 <https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1186.md>`_.
+
+----------
+Parameters
+----------
+
+1. ``String`` 20 Bytes:  The Address of the account or contract.
+2. ``Number[] | BigNumber[] | BN[] | String[]`` 32 Bytes: Array of storage-keys which should be proofed and included. See :ref:`web3.eth.getStorageAt <eth-getStorageAt>`.
+3. ``Number | String | BN | BigNumber``: Integer block number. Pre-defined block numbers as ``"latest"``, ``"earlist"``, ``"pending"``, and ``"genesis"`` can also be used.
```

Comment:
```suggestion
3. ``Number | String | BN | BigNumber``: Integer block number. Pre-defined block numbers as ``"latest"``, ``"earliest"``, ``"pending"``, and ``"genesis"`` can also be used.
```

In JSON-RPC docs for _getProof it says 
```
QUANTITY|TAG - integer block number, or the string "latest" or "earliest", see the default block parameter
```
Is this discrepancy about whether 'pending' is valid ignorable? I would guess yes but just checking....

Reply:
Oh, thanks! I only copied the sentence from above. I will fix the typo here and in all other places (earlist). Thanks!